mne_connectivity/decoding/decomposition.py

```diff
@@ -465,3 +469,306 @@ def get_transformed_indices(self):
+        data (a.k.a. the backward model). :footcite:`HaufeEtAl2014`.
+
+        Seed and target filters are plotted separately.
+
+        Parameters
+        ----------
+        %(info_decoding_plotting)s
+        %(components_topomap)s
+        %(ch_type_topomap)s
+        %(scalings_topomap)s
+        %(sensors_topomap)s
+        %(show_names_topomap)s
+        %(mask_filters_topomap)s
+        %(mask_params_topomap)s
+        %(contours_topomap)s
+        %(outlines_topomap)s
+        %(sphere_topomap)s
+        %(image_interp_topomap)s
+        %(extrapolate_topomap)s
+        %(border_topomap)s
+        %(res_topomap)s
+        %(size_topomap)s
+        %(cmap_topomap)s
+        %(vlim_topomap)s
+        %(cnorm_topomap)s
+        %(colorbar_topomap)s
+        %(colorbar_format_topomap)s
+        %(units_topomap)s
+        %(axes_topomap)s
+        %(name_format_topomap)s
+        %(nrows_topomap)s
+        %(ncols_topomap)s
+        %(show)s
+
+        Returns
+        -------
+        %(figs_topomap)s
+        """
+        if self.filters_ is None:
+            raise RuntimeError(
+                "no filters are available, please call the `fit` method first"
+            )
+
+        return self._plot_filters_patterns(
+            self.filters_,
+            info,
+            components,
+            ch_type,
+            scalings,
+            sensors,
+            show_names,
+            mask,
+            mask_params,
+            contours,
+            outlines,
+            sphere,
+            image_interp,
+            extrapolate,
+            border,
+            res,
+            size,
+            cmap,
+            vlim,
+            cnorm,
+            colorbar,
+            cbar_fmt,
+            units,
+            axes,
+            name_format,
+            nrows,
+            ncols,
+            show,
+        )
+
+    def _plot_filters_patterns(
+        self,
+        plot_data,
+        info,
+        components,
+        ch_type,
+        scalings,
+        sensors,
+        show_names,
+        mask,
+        mask_params,
+        contours,
+        outlines,
+        sphere,
+        image_interp,
+        extrapolate,
+        border,
+        res,
+        size,
+        cmap,
+        vlim,
+        cnorm,
+        colorbar,
+        cbar_fmt,
+        units,
+        axes,
+        name_format,
+        nrows,
+        ncols,
+        show,
+    ):
+        """Plot filters/targets for components."""
+        # Sort inputs
+        _validate_type(info, Info, "`info`", "mne.Info")
+        if components is None:
+            components = np.arange(self.n_components)
+
+        # plot seeds and targets
+        figs = []
+        for group_idx, group_name in zip([0, 1], ["Seeds", "Targets"]):
+            # create info for seeds/targets
+            group_info = deepcopy(info)
+            group_info = pick_info(group_info, self.indices[group_idx], copy=False)
```

Comment:
`mne.Info` overrides the default `dict.copy` to do a deepcopy so you could do
```
            group_info = info.copy()
            group_info = pick_info(group_info, self.indices[group_idx], copy=False)
```
but even this can be made more compact just by using the default `copy=True` of `pick_info`
```suggestion
            group_info = pick_info(info, self.indices[group_idx])
```

Reply:
Done!